.travis.yml

```diff
@@ -12,8 +9,10 @@ rvm:
   - ruby-head
   - rbx-2
 
-install:
-  - bundle install --retry=3
+install: bundle install --path=vendor/bundle --retry=3 --jobs=3
```

Comment:
The reason it wasn't caching was because the bundle install command was explicitly defined, so it overrode whatever default travis defines. I fixed this by setting a dir to install into, then explicitly telling travis to cache that dir :smile: